### ISSUE TITLE
[Breadcrumb] "..." button don't have a accessible name (WCAG2AA)

### ISSUE THREAD

**Issue by JoaoEnesGoncalves** [lumada-design/hv-uikit-react#893]
❗️  similar as https://github.com/pentaho/hv-uikit-react/issues/678

![3dots](https://user-images.githubusercontent.com/9842927/70229561-5dfd7100-174e-11ea-9f51-9f0d8d764037.PNG)

**To Reproduce**
```
pa11y --runner htmlcs --runner axe --standard WCAG2AA --root-element "div[class|='Component-content']" http://localhost:9002/iframe.html?id=corebreadcrumb--breadcrumb2
```
**Error Result**

```
Results for URL: http://localhost:9002/iframe.html?id=corebreadcrumb--breadcrumb2

 • Error: This element has role of "button" but does not have a name available to an accessibility API. Valid names are: element content, aria-label undefined, aria-labelledby undefined.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Div.Name
   ├── #root > div:nth-child(2) > div > div > div > div:nth-child(2) > div:nth-child(1) > div
   └── <div role="button" tabindex="0" class="DropDownMenu-icon-187"><div class="MoreOptionsHorizont...</div>

 • Error: Buttons must have discernible text (https://dequeuniversity.com/rules/axe/3.4/button-name?application=axeAPI)
   ├── button-name
   ├── #root > div:nth-child(2) > div > div > div > div:nth-child(2) > div:nth-child(1) > div
   └── <div role="button" tabindex="0" class="DropDownMenu-icon-187"><div class="MoreOptionsHorizont...</div>

2 Errors

```
